packages/ses-types-test/test/test-ses.js

```diff
@@ -0,0 +1,17 @@
+/// <reference types="ses"/>
+
+import test from 'ava';
+
+lockdown();
+
+test('harden', t => {
+  const { x, y } = harden({ x: 4, y: 3 });
```

Comment:
I don't see how this would fail if `harden` got a type of `any`. Help?

Reply:
Huh, good point. It doesn’t fail if I remove the reference. Will have to phone a friend.

Reply:
You could probably negate the test.
```suggestion
  // @ts-expect-error
  const { z } = harden({ x: 4, y: 3 });
```

Reply:
Do we still need this file?